conf/messages.en

```diff
@@ -1756,6 +1756,11 @@ raRAdjustmentsCYA.heading = raRAdjustmentsCYA
 raRAllowancesCYA.title = raRAllowancesCYA
 raRAllowancesCYA.heading = raRAllowancesCYA
 
+incomeSectionFinished.title = Have you finished this section?
```

Comment:
Do we need to duplicate this for every section? It seems to be common text for all sections.

Reply:
Tidy up the code and removed all duplicates for Have you finished this section pages